### PR TITLE
Add NFT quote and rpc messages

### PR DESCRIPTION
**Pull request by itsthetombomb**

### CODE REVIEW

File: api/bridge/definition.proto [hunk 1]
```diff
@@ -18,6 +20,18 @@ message BridgeProto {
     string chain_name = 4;
 }
 
+message BridgeQuoteProto {
```
Comment (ngeib) [inline api/bridge/definition.proto]: Should these messages follow the ...Request and ...Response convention?

File: api/bridge/definition.proto [hunk 2]
```diff
@@ -39,4 +53,12 @@ message WithdrawItemRequest {
 
 /* Get Bridge Call */
 message GetBridgeRequest {
+}
+
+/* Get Bridge Quote Call */
+message GetBridgeQuoteRequest {
```
Comment (ngeib) [inline api/bridge/definition.proto]: I recommend following using the same names as in the saga-proto repo.

File: api/bridge/rpc.proto [hunk 3]
```diff
@@ -13,4 +13,6 @@ service BridgeService {
     rpc WithdrawItem(saga.api.bridge.WithdrawItemRequest) returns (saga.common.ReceivedResponse);
     /* Get Bridge */
     rpc GetBridge(saga.api.bridge.GetBridgeRequest) returns (saga.api.bridge.BridgeProto);
+    /* Get Bridge Quote */
+    rpc GetBridgeQuote(saga.api.bridge.GetBridgeQuoteRequest) returns (saga.api.bridge.BridgeQuoteProto);
```
Comment (ngeib) [inline api/bridge/rpc.proto]: I recommend following using the same names as in the saga-proto repo.

Comment (itsthetombomb) [reply api/bridge/rpc.proto]: resolved

File: api/bridge/definition.proto (hunk 2 above)
Comment (itsthetombomb) [reply api/bridge/definition.proto]: resolved

**Review by ngeib — APPROVED** [review]

**Pull request merged**